internal/agent.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package internal
+
+import (
+	"net/url"
+	"os"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+)
+
+// AgentURLFromEnv determines the trace agent URL from environment variable
+// DD_TRACE_AGENT_URL. If the determined value is valid and not a UDS socket,
+// it returns the value and false. If the determined value is valid and a UDS
+// socket, it returns the UDS path and true. If the value is not valid, it returns
+// an empty string and false.
```

Comment:
This doc comment appears to be out of date, since `AgentURLFromEnv` now just returns a URL and not a bool.

Reply:
Nice catch! Updated 👍